src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/ccd/domain/RegionalProcessingCenter.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.sscscorbackend.service.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
```

Comment:
probably want @JsonIgnoreProperties(ignoreUnknown = true) to protect code from changes

Reply:
This has been fixed no idea why the comment has not been marked as done